src/logic/operate.js

```diff
@@ -0,0 +1,27 @@
+import Big from 'big.js';
+
+const Operate = (numberOne, numberTwo, operation) => {
+  const nOne = Big(numberOne);
+  const nTwo = Big(numberTwo);
+  switch (operation) {
+    case '+':
+      return (nOne.plus(nTwo)).toString();
+    case '-':
+      return (nOne.minus(nTwo)).toString();
+    case 'x':
+      return (nOne.times(nTwo)).toString();
+    case '÷':
+      if (nOne !== 0 && nTwo !== 0) {
+        return (nOne.div(nTwo)).toString();
+      }
+      return '0';
+    case '%':
+      if (nOne !== 0 && nTwo !== 0) {
+        return ((nOne.div(100)).times(nTwo)).toString();
```

Comment:
I think you need to turn the number into a percentile here and the only way to do that is to divide by 100, no need multiplying with the second number. percentage works with just one input 
![see](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/performOperate.png)